lib/sneakers/worker.rb

```diff
@@ -72,6 +72,8 @@ def process_work(delivery_info, metadata, msg, handler)
           end
           res = block_to_call.call(deserialized_msg, delivery_info, metadata, handler)
         end
+      rescue *opts[:raise_exceptions] => ex
+        raise
```

Comment:
Wouldn't it make sense to re-raise existing exception?

```suggestion
        raise ex
```